release/RELEASE.md

```diff
@@ -76,9 +76,11 @@ patches we backport to the 1.0 release branch).
    >
    > [ad-hoc]: https://github.com/DACH-NY/daml-language-ad-hoc
    >
-   > `ad-hoc.sh` prints IP address, username and password for the created Windows VM.
+   > `ad-hoc.sh create windows` prints IP address, username and password for the created Windows VM.
    > Save this output. You will need this information later when you create an RDP connection.
-   >
+   > 
+   > ‼️ After connecting, it's going to take some time for the machine to be configured (see notes below).
```

Comment:
The configuration of the machine starts immediately on startup, it doesn't wait for you to connect.

So if you create the machine, wait for fifteen minutes, and then connect, it should be ready to go. (Most of the time. Maybe plan for 20 minutes if you want to be sure.)

Reply:
Valid point, thanks for the good catch!